lib/dor/text_extraction/ocr.rb

```diff
@@ -11,6 +11,11 @@ def initialize(params = {})
         @workflow_context = params[:workflow_context]
       end
 
+      def abbyy_output_path
+        bare_druid = @cocina_object.externalIdentifier.split(':')[-1]
+        File.join(Settings.sdr.abbyy.local_output_path, bare_druid)
+      end
+
```

Comment:
While you are making this change, can you add an analogous  `abbyy_input_path` method?  This will come in handy in the as yet unwritten cleanup robot, and we can then refactor the`fetch-files` robot to make use of it (here:  https://github.com/sul-dlss/common-accessioning/blob/main/lib/robots/dor_repo/ocr/fetch_files.rb#L34-L38)

Reply:
Sounds good. Done. I am assuming you want to do this refactor in a new PR and not this one?

Reply:
Yeah, we can do that in a new PR.  I can ticket it once this is in.